src/ethernet_interface.cpp

```diff
@@ -471,6 +471,13 @@ void EthernetInterface::disableDHCP(IP::Protocol protocol)
 ObjectPath EthernetInterface::ip(IP::Protocol protType, std::string ipaddress,
                                  uint8_t prefixLength, std::string)
 {
+    // TODO This is a workaround to avoid IPv4 static and DHCP IP address
+    // coexistence Disable IPv4 DHCP while configuring IPv4 static address
+    if ((protType == IP::Protocol::IPv4) && dhcpIsEnabled(protType, false))
```

Comment:
How about static IPv6 configuration? Should the same kind of check be there for v6 as well?

Reply:
IPv6 addresses can co exist and both ipv6 static and dhcpv6 addresses work